engine/src/main/resources/assets/shaders/newParticle_frag.glsl

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2015 Benjamin Glatzel <benjamin.glatzel@me.com>
```

Comment:
Feel free to replace these older license statements with the latest one - even can go to 2017 now!

Same thing if you come across any `@author` tags - we don't use them anymore.